### DISCUSSION TITLE
能否合并另一个库的《旧播放页》功能

### DISCUSSION THREAD

**Discussion by chuyueye** [the1812/Bilibili-Evolved#3571] [功能建议 / Ideas]
我很喜欢另个库的css样式，能否加入这个feature呢
https://github.com/MotooriKashin/Bilibili-Old
这个库

**Comment by the1812** [comment]
直接装这个不就行了, 为啥非要合并

**Reply by Geeyun-JY3** [reply]
依稀记得那边开发者说是不兼容的（毕竟为了复原旧版页面好像做了不少改动
